Options.py

```diff
@@ -803,7 +811,7 @@ def verify(self, world: typing.Type[World], player_name: str, plando_options: "P
 
 
 class OptionDict(Option[typing.Dict[str, typing.Any]], VerifyKeys, typing.Mapping[str, typing.Any]):
-    default: typing.Dict[str, typing.Any] = {}
+    default = {}
```

Comment:
I get that the default value was not changed, only the type removed, but shouldn't that default default value be something immutable? For safety measure.

Reply:
It has been considered and discussed in the past. If it were as simple and straightforward as `OptionList` and `OptionSet`, I think it would have been done before.
But yeah, that's out of scope for this PR.